core/src/test/java/io/questdb/std/DirectLongListTest.java

```diff
@@ -27,13 +27,77 @@
 import io.questdb.cairo.BinarySearch;
 import io.questdb.log.Log;
 import io.questdb.log.LogFactory;
+import io.questdb.test.tools.TestUtils;
 import org.junit.Assert;
 import org.junit.Test;
 
+import java.util.Arrays;
+
 public class DirectLongListTest {
 
     private static final Log LOG = LogFactory.getLog(DirectLongListTest.class);
 
+    @Test
+    public void test128BitSort() throws Exception {
+        TestUtils.assertMemoryLeak(() -> {
+            try (DirectLongList list = new DirectLongList(256, MemoryTag.NATIVE_LONG_LIST)) {
+                final int N = 100;
+                for (int i = 0; i < N; ++i) {
+                    list.add((100 - i - 1) / 10);
+                    list.add((100 - i - 1));
+                }
+
+                Vect.sort128BitAscInPlace(list.getAddress(), list.size() / 2);
+
+                for (int i = 0; i < 100; i++) {
+                    Assert.assertEquals(i / 10, list.get(i * 2));
+                    Assert.assertEquals(i, list.get(i * 2 + 1));
+                }
+            }
+        });
+    }
+
+    @Test
+    public void test128BitSortFuzzTest() throws Exception {
+        long s0 = System.currentTimeMillis();
+        long s1 = System.nanoTime();
+        Rnd rnd = new Rnd(s0, s1);
+        System.out.printf("random seed : %d, %d%n", s0, s1);
```

Comment:
nit: could use `LOG` here

Reply:
done in https://github.com/questdb/questdb/pull/1778/commits/3b8609ba3442fa0a590cd19693726b231d02cbff